packages/bot-skeleton/src/scratch/blocks/Math/math_number.js

```diff
@@ -35,7 +35,7 @@ Blockly.Blocks.math_number = {
         };
     },
     numberValidator(input) {
-        if (/^-?([0][.]|[1-9]+[.])?([0]|[1-9])*$/.test(input)) {
+        if (/^([0][.]|[0-9]+[.]?){1,}$/.test(input)) {
```

Comment:
why this block doesn't support negative number anymore?

Reply:
Good catch, I've updated the patterns.